src/utils/record-completeness/ComputeRecordCompleteness.ts

```diff
@@ -19,6 +19,10 @@ export function computeRecordCompleteness(vehicle: Vehicle): string {
     throw new HTTPError(400, ERRORS.SYSTEM_NUMBER_GENERATION_FAILED);
   }
 
+  if (vehicle.techRecord[0].hiddenInVta) {
```

Comment:
Is this method only called on the PUT updateTechRecords endpoint? Just wondering if there's ever an instance where techRecord has a length greater than 1 and this hard coded indexing might cause issues?

Reply:
All utility methods use the same approach to get the `techRecord` so I followed suit.

This is called on POST and PUT.

Reply:
Sure, I posted this PR on the `Code Reviews` channel.

Reply:
Ignore my comment above. I was thinking from a GET point of view, which this is not the case.